.vscode/autogpt-workspace.code-workspace

```diff
@@ -0,0 +1,69 @@
+{
+  "folders": [
+    {
+      "name": ".github",
+      "path": "../.github"
+    },
+    {
+      "name": ".vscode",
+      "path": "../.vscode"
+    },
+    {
+      "name": "assets",
+      "path": "../assets"
+    },
+    {
+      "name": "autogpt",
+      "path": "../autogpt"
+    },
+    {
+      "name": "benchmark",
+      "path": "../benchmark"
+    },
+    {
+      "name": "docs",
+      "path": "../docs"
+    },
+    {
+      "name": "forge",
+      "path": "../forge"
+    },
+    {
+      "name": "frontend",
+      "path": "../frontend"
+    },
+    {
+      "name": "autogpt_server",
+      "path": "../rnd/autogpt_server"
+    },
+    {
+      "name": "root_folder",
+      "path": ".."
+    }
+  ],
+  "settings": {
+    "python.analysis.useLibraryCodeForTypes": true,
+    "python.analysis.autoImportCompletions": true,
+    "python.analysis.typeCheckingMode": "basic",
+    "python.analysis.completeFunctionParens": true,
+    "python.analysis.inlayHints.functionReturnTypes": true,
+    "python.analysis.inlayHints.variableTypes": true,
+    "python.analysis.autoFormatStrings": true,
+    "python.analysis.inlayHints.callArgumentNames": "partial",
+    "python.analysis.fixAll": [
+      "source.convertImportFormat",
+      "source.unusedImports"
+    ]
+  },
+  "extensions": {
+    "recommendations": [
+      "charliermarsh.ruff",
+      "dart-code.flutter",
+      "ms-python.black-formatter",
```

Comment:
Ruff + Black?

Reply:
Some use ruff some use black

Reply:
It suggests installing both because we lack consistency. They are both needed if you want to develop our projects. Our inconsistency won't be solved in this issue, so this should be non blocking, and removed once we have reached consistency